Tmain/readtags-alias-for-boolean.d/run.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Copyright: 2020 Masatake YAMATO
+# License: GPL-2
+
+READTAGS=$3
+#V="valgrind --leak-check=full --track-origins=yes -v"
+V=
+
+. ../utils.sh
+
+if ! [ -x "${READTAGS}" ]; then
+	skip "no readtags"
+fi
+
+if ! ( "${READTAGS}" -h | grep -q -e -Q ); then
+    skip "no qualifier function in readtags"
+fi
+
+echo '!_ -Q #t point3d'
+${V} ${READTAGS} -t output.tags -Q '#t' point3d
+
+echo '!_ -Q ture point3d'
```

Comment:
```suggestion
echo '!_ -Q true point3d'
```

There are similar mistakes in other places :) But I wonder how the test passed since you spell it right in the code.

Reply:
Adding the comments after writing the test code.
Thank you!

Reply:
It seems that I took a mistake in reflecting the suggestions.